### ISSUE TITLE
Feature: Auto-create for extended tables

### ISSUE THREAD

**Issue by gaumondp** [clickstorm/cs_seo#229]
Maybe it's not possible or Core-related but it would be great if ext:News record for instance get their SEO options created automatically.

Right now you have to "Create new" in the SEO tab for all news records.

It's a minor annoyance but would be appreciated by editors.

**Comment by mhirdes** [comment]
It would be possible. But then every table would have it's own seo fields. Also their has to be a migration and the queries has to change. That would be a request for a major release. Maybe for TYPO3 v10.

**Comment by mhirdes** [comment]
Not really. It's an IRRE relation. I don't know a solution to immediatly create a new irre record, if the parent record is created.
